bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/StateMachine.java

```diff
@@ -18,10 +18,10 @@
 
 package io.ballerina.runtime.internal;
 
+import io.ballerina.identifier.Utils;
 import io.ballerina.runtime.api.creators.ErrorCreator;
 import io.ballerina.runtime.api.utils.StringUtils;
 import io.ballerina.runtime.api.values.BError;
-import org.apache.commons.lang3.StringEscapeUtils;
```

Comment:
`org.apache.commons.lang3.StringEscapeUtils` Class is used in multiple other packages as well. Since we have `Utils` implementation for the same purpose we can replace them as well. Shall we do that?

Reply:
Sounds good to me!